src/fluid/fluid.h

```diff
@@ -25,9 +25,8 @@
 #include <cstdlib>    // std::size_t
 #include <exception>  // std::exception_ptr
 #include <iostream>   // std::cout, std::endl
-#include <vector>     // std::vector
-
 #include <sycl/sycl.hpp>
+#include <vector>  // std::vector
```

Comment:
This is wrong, I don't know why it's doing this. It should allow for spaces between the include blocks

Reply:
Resolved by adding our own `.clang-format` style file based on the Google style. The new style overrides the include categories with a separate category for SYCL includes.